common/test/acceptance/tests/helpers.py

```diff
@@ -199,6 +199,20 @@ def enable_css_animations(page):
     """)
 
 
+def coordinates_for_scrolling(page, css_selector):
+    """
+    Get the x and y coordinates of elements
+    """
+    coordinates_dict = page.browser.find_element_by_css_selector(css_selector)
```

Comment:
Thinking further on this, I think it'd be better to use bok-choy's query methods for finding the object. Then you'd have the built-in protections for checking first that you're on the right page, etc.

E.g. maybe something like this:
```
try:
	elem = page.q(css=css_selector).first.results[0]
except IndexError:
	raise SomethingSayingTheObjectIsntThere

x_axis = elem.location.get('x')
y_axis = elem.location.get('y')
return x_axis, y_axis
```